include/tesseract/serialis.h

```diff
@@ -43,14 +41,17 @@ constexpr size_t countof(T const (&)[N]) noexcept {
   return N;
 }
 
-// Function to read a GenericVector<char> from a whole file.
+// Function to read a std::vector<char> from a whole file.
```

Comment:
Probably we should replace `vector<char>` with `vector<uint8_t>` for binary data later.